lib/vdc/DataMgr.cpp

```diff
@@ -635,6 +644,40 @@ int DataMgr::Initialize(const vector<string> &files, const std::vector<string> &
         SetErrMsg("Failed to get time coordinates");
         return (-1);
     }
+
+#if DCP_ENABLE_PARTICLE_DENSITY
```

Comment:
Is this just test code? Are we planning on enabling it? Need to handle error returns below if so. Also, it looks like the position_x variable is hardcoded. Is that reasonable?

Reply:
This is prototype code for automatically generating particle density fields when users load particle data. It is in a stage where if you set DCP_ENABLE_PARTICLE_DENSITY it will work but it will have a fixed grid size and it uses a sub-optimal sampling algorithm.

Reply:
Could you add a comment to that effect?

Reply:
Done.